stac_fastapi/types/setup.py

```diff
@@ -0,0 +1,53 @@
+"""arturo-stac-api."""
+import os
+from imp import load_source
+
+from setuptools import find_namespace_packages, setup
+
+with open("README.md") as f:
+    desc = f.read()
+
+# Get version from stac-fastapi-api
+__version__ = load_source(
+    "stac_fastapi.types.version",
+    os.path.join(os.path.dirname(__file__), "stac_fastapi/types/version.py"),
+).__version__  # type:ignore
```

Comment:
Might be simpler to have a `version.txt` file at the repo root, or maybe consider something like [`versioneer`](https://github.com/python-versioneer/python-versioneer)